docs/jwst.rst

```diff
@@ -54,14 +54,31 @@ All classes share some common attributes:
     available modes well enough to be aware if they are trying to
     simulate an inconsistent or physically impossible configuration. 
 
+Optical Telescope Element (OTE)
+===============================
+
+The JWST Optical Telescope Element consists of the telescope optics that serve all the science instruments and the fine guidance sensor. Most notably, this means the primary, secondary, and tertiary mirrors. The OTE contributes to the overall wavefront error (and therefore the aberrations in instrument PSFs) in a few ways:
+
```

Comment:
> any contributions from the pickoffs are also included in the OPD right?)

Not explicitly. I suppose implicitly the pickoff folds can contribute to the OTE to ISIM misalignments but I don't know if it's actually tracked as part of that budget. I doubt it. The pickoff mirrors were of course used to send light into each SI during the ISIM CV testing and so their contributions are most definitely part of the SI WFE models. 

But in practice the pickoff mirrors should have a negligible contribution, for two reasons. First, flat mirrors like those can be made very high quality. Second, even more fundamentally the pickoffs are all near focus, and phase errors error near a focus don't impact the PSF intensity in a subsequent focal plane. They just change the phase of that PSF, which we don't care about.